Sprint-1/implement/dedupe.test.js

```diff
@@ -13,15 +13,41 @@ E.g. dedupe([1, 2, 1]) target output: [1, 2]
 
 // Acceptance Criteria:
 
-// Given an empty array
-// When passed to the dedupe function
-// Then it should return an empty array
-test.todo("given an empty array, it returns an empty array");
+describe("dedup Tests", () => {
+  // Given an empty array
+  // When passed to the dedupe function
+  // Then it should return an empty array
+  test("Given an empty array, it returns an empty array", () => {
+    const input = dedupe([]);
+    const expected = [];
+    expect(input).toEqual(expected);
+  });
 
-// Given an array with no duplicates
-// When passed to the dedupe function
-// Then it should return a copy of the original array
+  // Given an array with no duplicates
+  // When passed to the dedupe function
+  // Then it should return a copy of the original array
+  test("Given an array with no duplicates, it returns a copy of the original array", () => {
+    const input = dedupe([7, 2, 8, 9]);
+    const expected = [7, 2, 8, 9];
+    expect(input === expected).toEqual(false);
```

Comment:
This does now check that the returned array isn't the same as the passed one (though `expect(input).not.toBe(expected)` may be more clear/idiomatic as its error message will be better on failure), but _doesn't_ check that the contents are the same any more. You may need two assertions to check both thing :)